content/guides/visualization.md

```diff
@@ -16,6 +16,24 @@ NeuroData maintains a Neuroglancer [fork](https://github.com/neurodata/neuroglan
 1. Export and load state to/from an external JSON server
 1. Segmentation overlays for ARA and Zbrain ontologies, at [ara.viz.neurodata.io](https://ara.viz.neurodata.io) and [zbrain.viz.neurodata.io](https://zbrain.viz.neurodata.io)
 
+### View Local Images in Neuroglancer Example
+
+The following example can be used to help you view your own images in neuroglancer. This example uses a [script]([url('/content/guides/upload.py')]) to generate a test image, then convert from tif to precomputed format. It will also require running a [script](https://github.com/google/neuroglancer/blob/master/cors_webserver.py)  from neuroglancer to host the local data:
+
+1. mkdir ./test_output
+1. python upload.py precomputed://file://test_output/
```

Comment:
I was confused as to what this step in the instructions means. I'm not sure if the "//" and ":" are things I have to add in or if they are just to separate directories in the path to the test outputs. Thankfully the error message from cloudvolume explained what format the cloud path should have, so I realized that I should copy the line from this step exactly. I would add some information about what "precomputed:" and "file:" mean with respect to neuroglancer

Reply:
Honestly, I'm not exactly sure myself, but does the new "help" info in upload.py make it more clear? I think the interested reader could check out the neuroglancer docs